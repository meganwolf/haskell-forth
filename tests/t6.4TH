25.0 5 * 9.0 + .
25.0 10 -  CR .
15.0 2 / CR .
10.0 2 ^ CR .
3.0 2 ^ 7 + CR .
92 10 / CR
