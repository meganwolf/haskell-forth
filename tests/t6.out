134.0
15.0
7.5
100.0
16.0

"Stack not empty"
[Integer 9]